### PR TITLE
RF | Methods return webElements in FolderConfigPage | _FolderTest

### PR DESCRIPTION
**Pull request by iukait**
In class FolderConfigPage renamed webElements:
-clickMetricsButton to healthMetricsButton
-clickAddMetricButton to addMetricButton
-clickMetricsItem to childItem
-getMetricElement to textChildItem
-deleteHealthMetric to deleteChildItemButton
renamed methods:
clickMetricsButton() to clickHealthMetricsButton()
clickMetricsItem() to clickChildItem()
getMetricElement() to isAddMetricDisplayed()
deleted deleteHealthMetric()
added clickDeleteChildItem()
Refactored tests: testAddHealthMetrics,testRemoveHealthMetrics

### CODE REVIEW

**Comment by iukait** [comment]
[RF | Methods return webElements](https://trello.com/c/o6lZDx7S/1256-rf-methods-return-webelements)

File: src/test/java/model/FolderConfigPage.java [hunk 1]
```diff
@@ -60,50 +59,51 @@ public FolderProjectPage saveConfigAndGoToFolderPage() {
...
+    public boolean isAddMetricDisplayed() {
+
+        return addMetricButton.isDisplayed();
+    }
+
+    public String getTexChildItem() {
+        textChildItem.getText();
 
-        return getMetricElement;
+        return textChildItem.getText();
```
Comment (Rom-T) [inline src/test/java/model/FolderConfigPage.java:102]: Строка 100 (textChildItem.getText();) не нужна, так как используется в return

File: src/test/java/model/FolderConfigPage.java [hunk 2]
```diff
@@ -60,50 +59,51 @@ public FolderProjectPage saveConfigAndGoToFolderPage() {
...
         return this;
     }
 
-    public WebElement getMetricElement() {
+    public boolean isAddMetricDisplayed() {
+
+        return addMetricButton.isDisplayed();
+    }
+
+    public String getTexChildItem() {
```
Comment (Rom-T) [inline src/test/java/model/FolderConfigPage.java:99]: getTexChildItem - > getTex**t**ChildItem

**Review by Rom-T — APPROVED** [review]

File: src/test/java/model/FolderConfigPage.java (hunk 1 above)
Comment (iukait) [reply src/test/java/model/FolderConfigPage.java]: спасибо, исправила

**Review by patriotby07 — APPROVED** [review]

**Review by azellus-tercia — APPROVED** [review]

**Pull request merged**